README.md

```diff
@@ -59,6 +59,12 @@ As a contributor, you might also need the following commands:
 
 - `tldr --render <path>` render a local page for testing purposes
 
+To view tldr pages for a different language, set an environment variable `LANG` containing a valid [POSIX locale](https://www.gnu.org/software/gettext/manual/html_node/Locale-Names.html#Locale-Names) (such as `zh`, `pt_BR`, or `fr`) and then run the above commands as usual. In most `*nix` systems, this variable will already be set.
```

Comment:
Maybe prefix this with information about what language is used by default. Something like this:

> tldr-pages defaults to showing pages in the current language of the operating system, or English if that's not available.

WDYT? (Assuming it accurately described the actual behavior, of course.)

Reply:
Sounds good.